### ISSUE TITLE
'Reset to Default Rooms' doesn't update rooms tabs without restart

### ISSUE THREAD

**Issue by bencresty** [opentoonz/opentoonz#4537]
### Description

When hitting Windows --> Workspace --> Reset to Default Rooms nothing visually happens. The same custom room tabs stay active and seem to even still work. No warning to restart or whatsoever.

Only after a restart of OpenToonz the default rooms tabs are shown instead of the custom setup.

This is obviously confusing as it seems like something went wrong and nothing happened. And is prone to errors as the visual state isn't matching the actual state?

### Steps to Reproduce

1. Change rooms tabs
2. Windows --> Workspace --> Reset to Default Rooms
3. See that the changed room tabs layout is still active. Nothing visually changed
4. Restart OpenToonz
5. Suddenly now there the default tabs are

### Expected Behavior

Immediately update the rooms in the GUI after resetting to default.

### OpenToonz Version

1.6

### OpenToonz Version Information

built Apr 4 2022 10:30:14

### Operating System

Windows

### GPU

NVIDIA

### Graphics Tablet

Wacom

**Comment by DarrenTAnims** [comment]
Yes, you have to restart to see the room reset. This has always bugged me too. I wished the command either showed a popup telling you to restart or the command was changed to "Reset rooms and restart OpenToonz" and then auto-restarted.

**Comment by justburner** [comment]
This was already solved in #4501

**Comment by RodneyBaker** [comment]
This seems to be a duplicate of previous request from #3911.
If there are differences please let us know.

**Comment by justburner** [comment]
I've linked to the PR to auto close when it gets merged.
